src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int COUTN_OBJECT = 5;
+    private static final Figure[] arrayOfFigures = new Figure[COUTN_OBJECT];

```

Comment:
As per task description:

> in the main() method we need to create an array of figures 


So, feel free to move array creation to the method.